src/lib.rs

```diff
@@ -132,6 +138,7 @@ pub fn plugin_registrar(reg: &mut Registry) {
         bit_mask::BAD_BIT_MASK,
         bit_mask::INEFFECTIVE_BIT_MASK,
         collapsible_if::COLLAPSIBLE_IF,
+        cyclomatic_complexity::CYCLOMATIC_COMPLEXITY,
```

Comment:
shouldn't that be `metrics::cyclomatic_complexity::CYCLOMATIC_COMPLEXITY`?

We haven't had any submodules so far in clippy.